README.md

```diff
@@ -6,13 +6,13 @@ You can also find an archive of samples targeting .NET 4.7.2 in the [netframewor
 
 The samples in this repo are generally about illustrating specific concepts and may go against accessibility best practices. However, the team has spent some time illustrating accessibility best practices in a subset of these samples.
 
-* [ExpenseItIntro](https://github.com/microsoft/WPF-Samples/tree/main/Getting%20Started/WalkthroughFirstWPFApp)
+* [ExpenseItIntro](https://github.com/microsoft/WPF-Samples/tree/main/Getting%20Started/WalkthroughFirstWPFApp/csharp)
 * [ExpenseItDemo](https://github.com/microsoft/WPF-Samples/tree/main/Sample%20Applications/ExpenseIt/ExpenseItDemo)
 * [DataBindingDemo](https://github.com/microsoft/WPF-Samples/tree/main/Sample%20Applications/DataBindingDemo)
 * [CustomComboBox](https://github.com/microsoft/WPF-Samples/tree/main/Sample%20Applications/CustomComboBox)
 * [EditingExaminerDemo](https://github.com/microsoft/WPF-Samples/tree/main/Sample%20Applications/EditingExaminerDemo)
 
-For additional WPF samples, see [WPF Samples](https://learn.microsoft.com/en-us/dotnet/desktop/wpf/wpf-samples?view=netframeworkdesktop-4.8).
+For additional WPF samples, see [here](https://github.com/Microsoft/WPF-Samples).
```

Comment:
It would be better if we point this to netframework branch here rather than the main repo.

Reply:
```suggestion
For WPF samples using .NET Framework, visit [here](https://learn.microsoft.com/en-us/dotnet/desktop/wpf/wpf-samples?view=netframeworkdesktop-4.8).
```

And update the link on that page to https://github.com/microsoft/WPF-Samples/tree/netframework